github/users.go

```diff
@@ -144,6 +144,22 @@ type UserListOptions struct {
 
 // ListAll lists all GitHub users.
 //
+// To paginate through all users, populate 'Since' with the ID of the last user.
+// Example (warning: this will quickly eat through your GitHub API quota!):
```

Comment:
My second comment is completely optional.

Have you considered using real Go examples here, instead of embedding it into the comment? E.g., https://godoc.org/testing#hdr-Examples. Would that be better or not?